src/pybamm/simulation.py

```diff
@@ -400,6 +401,9 @@ def solve(
             Whether to show a progress bar for cycling. If true, shows a progress bar
             for cycles. Has no effect when not used with an experiment.
             Default is False.
+        t_interp : None, list or ndarray, optional
```

Comment:
`t_interp` is not passed to  `solver.solve` for the case `self.operating_mode in ["without experiment", "drive cycle"]`

Reply:
also, does the docstring for t_eval need to be updated? It currently says "Can be provided as an array of times at which to return the solution", but this is the role of `t_interp` now.